src/server/pages/user-page.jsx

```diff
@@ -62,7 +66,14 @@ export default class UserPage extends React.Component {
                 alt={`${thisUser.name}'s avatar`}
               />
               <h1>{thisUser.full_name}</h1>
-              <h2>{thisUser.name}</h2>
+              <h2>
+                {thisUser.name}{" "}
+                {thisUser.github_url && (
+                  <a href={thisUser.github_url}>
```

Comment:
(1) mind if we reverse the order so it is icon then name? That'd require a right-ad on the icon of about 5px
(2) can we set this h2 `font-size` to `1em`? It feels a bit big, and I imagine down the line we may want to include other things like this
(3) could we have the anchor wrap the text as well, not just the icon?
(4) can the anchor come with a `target='_blank'`?

These are all a matter of taste, so feel free to take it or leave the feedback!

![screen shot 2019-02-27 at 1 29 16 pm](https://user-images.githubusercontent.com/95735/53524224-b6069900-3a93-11e9-9b1d-18e2b2b7d911.png)

Reply:
this all sounds good to me, i'll try to change things as you suggest